### PR TITLE
clusterapi: Add 'watch' verb to scale-from-zero example

### PR DESCRIPTION
**Pull request by dlipovetsky**
#### What type of PR is this?

/kind documentation

#### What this PR does / why we need it:
Adds the 'watch' verb to the RBAC example. If the 'get' and 'list' verbs are present, but the 'watch' verb is absent, the autoscaler reports an error. For example:

```log
cluster-autoscaler-b8949d8b9-76vcd E1006 22:11:43.056176       1 reflector.go:148] k8s.io/client-go/dynamic/dynamicinformer/informer.go:108: Failed to watch infrastructure.cluster.x-k8s.io/v1beta2, Resource=vcdmachinetemplates: unknown
```

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: cluster-autoscaler/cloudprovider/clusterapi/README.md [hunk 1]
```diff
@@ -254,6 +254,7 @@ rules:
     verbs:
     - get
     - list
+    - watch
```
Comment (Shubham82) [inline cluster-autoscaler/cloudprovider/clusterapi/README.md:257]: @dlipovetsky If you add this, then you have to update (add `watch` verb) at line no.:242 such as:

> need permission to `get`, `list`, and `watch` the infrastructure machine templates for

Comment (dlipovetsky) [reply cluster-autoscaler/cloudprovider/clusterapi/README.md:257]: Thanks, I missed that. Fixed and force-pushed.

**Review by elmiko — COMMENTED** [review]
thanks @dlipovetsky !

/approve
/lgtm

**Pull request merged**